strings/tips/marine.txt

```diff
@@ -85,21 +84,20 @@ As an engineer, placing turrets on open spaces and without defenses is a poor ch
 As a corpsman, the medevac stretcher needs to be linked to an active, powered beacon for it to teleport the patient.
 As a corpsman, the marines are counting on you to fix them up.
 As a corpsman, scan before you treat and you can't be beat.
-As a corpsman or doctor, remember that your HUD will tell you how much time is left to defibrillate a patient. From highest to lowest: Green, orange and red.
+As a corpsman or doctor, remember that your HUD will tell you how much time is left to defibrillate a patient. From highest to lowest: yellow, orange and red.
 As a corpsman, remember to bring extra kelotane or dermaline to heal burn damage. Xenos love acid and burning marines.
-As a corpsman, hypervene is useful. Use it to purge deadly neurotoxins and larva boosting growth hormones.
-The B18 armor from Requisitions has an built-in medical system. The Valkyrie Automedical Armor System can do the same for the XM-02 Combat Exoskeleton.
-The V1 tarp (available from Requisitions) can be stealted once you covered yourself with it.
+As a corpsman, hypervene is useful. Use it to purge deadly neurotoxins and larva-boosting growth hormones.
+The B18 armor from Requisitions has an built-in medical system. The Valkyrie Automedical Armor System can do the same for other armors.
+The V1 tarp (available from Requisitions) will blend you into the tile you're on once you cover yourself with it.
 The FL-84 flamethrower has an inbuilt fire extinguisher as an attachment.
 As the PO, pay attention to the radio! Who knows if someone needs immediate surgery or evacuation until their fate becomes worse?
 As the Captain, you have a unique Mateba revolver that takes a chunk of a target's healthpool down. However, you are important for completing the mission, so do not participate in active combat using the Mateba!
 Even if a patient dies a second after being successfully revived by a defibrillator, this resets the time it takes for them to become permanently brain dead.
 Don't forget that xenomorphs spawn infinitely during the Crash gamemode. Don't waste too much time hunting xenomorphs down!
-The AI and synthetics can talk together on their own channel using the :n key.
-You can memorialize fallen marines by using their dogtags on the ship's memorial! Memorialized solders will be displayed at the end of the round.
```

Comment:
?

Reply:
will fix the :n --> .n back to the :n.

unless you meant the memorial thing; that hasn't worked in months.